lib/resolve.ts

```diff
@@ -44,10 +43,12 @@ export function resolveTask(
 	.then((res: Resolve.ResolvedBundle) => {
 		task.projectType = res.projectType;
 		task.buildStream = res.tarStream;
+		task.resolved = true;
 		return task;
 	})
-	.catch((e: Error) => {
-		throw new ProjectResolutionError(e);
+	.catch(() => {
+		task.projectType = 'Could not be detected';
```

Comment:
Doesn't need `task.resolved = false;` here as well?

Reply:
It's not strictly necessary due to the flag having to be defined from the start: https://github.com/resin-io-modules/resin-multibuild/pull/15/files#diff-37a775e32e4a1ab95c3a6ada682775dcR45 

If you think it makes things clearer though I'm happy to add it in?

Reply:
Ah, I see, no then that's fine.